src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -36,10 +36,12 @@ public InputOutputColumnPair(string outputColumnName, string inputColumnName = n
         }
 
         [BestFriend]
-        internal static (string outputColumnName, string inputColumnName)[] ConvertToValueTuples(InputOutputColumnPair[] infos)
-        {
-            return infos.Select(info => (info.OutputColumnName, info.InputColumnName)).ToArray();
-        }
+        internal static (string outputColumnName, string inputColumnName)[] ConvertToValueTuples(InputOutputColumnPair[] infos) =>
+            infos.Select(info => (info.OutputColumnName, info.InputColumnName)).ToArray();
+
+        [BestFriend]
+        internal static  InputOutputColumnPair[] ConvertFromValueTuples((string outputColumnName, string inputColumnName)[] infos) =>
+            infos.Select(v => new InputOutputColumnPair(v.outputColumnName, v.inputColumnName)).ToArray();
```

Comment:
>v [](start = 25, length = 1)

nit: could be a good thing if we reconcile with the above `info => ...` instead of `v => ...`